### PR TITLE
MARKET-1331 Skeleton

### PR DESCRIPTION
**Pull request by StasivPavlo**
image
image

### CODE REVIEW

File: components/bl-skeleton-component/component.json [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        {
+          "value": "rounded",
+          "label": "Rounded"
+        }
+      ]
+    },
+    {
+      "type": "number",
+      "name": "width",
+      "label": "Width(px)",
```
Comment (v-excelsior) [inline components/bl-skeleton-component/component.json:41]: I assume that property should accept any values, for example 100%

**Review by v-excelsior — CHANGES_REQUESTED** [review]

File: components/bl-skeleton-component/src/index.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  const { style, display, classList, variant, width, height, animation } = component;
+
+  if (!display) {
+    return null;
+  }
+
+  return (
+    <div
+      className={ cn('bl-customComponent-skeleton', `${ variant }`, `${ animation }`, classList) }
+      style={ { ...style, width: width ? width : '100%', height: height ? height : '100%' } }>
```
Comment (Valodya) [inline components/bl-skeleton-component/src/index.js:13]: `width: width || '100%'`

File: components/bl-skeleton-component/src/index.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+export default function Skeleton({ component }) {
+  const { style, display, classList, variant, width, height, animation } = component;
+
+  if (!display) {
+    return null;
+  }
+
+  return (
+    <div
+      className={ cn('bl-customComponent-skeleton', `${ variant }`, `${ animation }`, classList) }
```
Comment (Valodya) [inline components/bl-skeleton-component/src/index.js:12]:
```
cn('bl-customComponent-skeleton',  variant, animation, classList)
```

File: components/bl-skeleton-component/README.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
+# Skeleton
```
Comment (Valodya) [inline components/bl-skeleton-component/README.md:1]: please correct the readme following rules of our etalon

**Comment by Valodya** [comment]
@StasivPavlo also please add a preview and a thumbnail image

**Pull request merged**